x-pack/test/api_integration/apis/uptime/rest/index.ts

```diff
@@ -44,6 +44,7 @@ export default function({ getService, loadTestFile }: FtrProviderContext) {
       loadTestFile(require.resolve('./dynamic_settings'));
       loadTestFile(require.resolve('./snapshot'));
       loadTestFile(require.resolve('./monitor_states_generated'));
+      loadTestFile(require.resolve('./snapshot'));
```

Comment:
it's already there..

Reply:
Merge mistake, whoops 😊

Reply:
Fixed in https://github.com/elastic/kibana/pull/63913/commits/7bc051ebf4194c944e7b1d70e0880524942b7e57.